### PR TITLE
Add node_id to the pull request model

### PR DESCRIPTION
**Pull request by ricardoamador**
Adding the node_id to the pull request will allow flutter to make one less call to the graphql api if we can collect this information from the event via a webhook.

### CODE REVIEW

File: lib/src/common/model/pulls.dart [hunk 1]
```diff
@@ -46,6 +47,10 @@ class PullRequest {
   /// Pull Request ID
   int? id;
 
+  /// Unique node identification string.
+  @JsonKey(name: 'node_id')
```
Comment (CaseyHillers) [inline lib/src/common/model/pulls.dart:51]: This isn't necessary as by default it's snake_case

Comment (ricardoamador) [reply lib/src/common/model/pulls.dart:51]: Ah okay. I will remove it.

**Review by CaseyHillers — APPROVED** [review]

**Comment by robrbecker** [comment]
I'll create a release with this in it ...

**Comment by robbecker-wf** [comment]
Released in version 9.13.0 https://github.com/SpinlockLabs/github.dart/releases/tag/9.13.0

**Comment by ricardoamador** [comment]
Perfect! Thank you so much guys!

**Pull request merged**